.github/workflows/deploy.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           **/pyproject.toml
     - name: Install dependencies part I
       run: |
-        sudo apt-get install dssp
+        sudo apt-get install dssp=3.0.0-3build1
```

Comment:
do we want to keep this?

Reply:
Meh. Don't particularly care either way, but probably not